### PR TITLE
update: speaker panel details

### PR DESCRIPTION
**Pull request by kateinkim**
# Description

Trello Card - 85-talk-details-include-multiple-speakers-for-panel

Summary of the changes:

- Updated Icon button style to match with design
- Created a separate screen for speaker panel details 
- Updated carousel card styles to show different styles in the speaker panel details screen

# Graphics

Icon Button Update
|Figma|Before|Updated|
|-|-|-|
|CleanShot 2023-02-10 at 13 28 38@2x|CleanShot 2023-02-10 at 13 30 34@2x|CleanShot 2023-02-10 at 13 27 45@2x|

Speaker Panel  Screen
| iOS            | Android            |
| -------------- | ------------------ |
|<video width="350" src="https://user-images.githubusercontent.com/53795920/218198726-3e5474a8-b70a-4e46-b11c-65fd82a7057e.mp4">|<video width="350" src="https://user-images.githubusercontent.com/53795920/218198809-422908ea-e963-40fc-9c5f-11d42d6f5691.mp4">|

# Checklist:

- [ ] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [ ] I have run tests and linter

### CODE REVIEW

**Comment by kateinkim** [comment]
[Talk details: include multiple speakers for panel](https://trello.com/c/jtkLuRd0/85-talk-details-include-multiple-speakers-for-panel)

File: app/components/Carousel.tsx [hunk 1]
```diff
@@ -115,14 +131,50 @@ export function Carousel(props: CarouselProps) {
             )
           }
 
+          let socialLinks = null
+          if (props.preset === "dynamic") {
+            const { socialLinks: socialLinksData } = item as DynamicCarouselItem
+            socialLinks = socialLinksData && (
+              <View style={$linksContainer}>
+                {socialLinksData.twitter && (
+                  <IconButton
```
Comment (robinheinze) [inline app/components/Carousel.tsx:140]: I think you could probably DRY this up a bit...maybe if you had a little custom component called `SocialButton` which takes URL and icon name as props?

Also you could avoid the three conditional renders by defining a little object like this:  
```
 const socialButtons = [
    { url: socialLinks.twitter, icon: "twitter" },
    { url: socialLinks.github, icon: "github" },
    { url: socialLinks.externalURL, icon: "link" },
  ]
```
then mapping over it and returning your custom component.

**Review by robinheinze — COMMENTED** [review]
I've only reviewed half so far, but I had one comment i figured I'd share since I might not get back to it until next week.

File: app/components/Carousel.tsx (hunk 1 above)
Comment (kateinkim) [reply app/components/Carousel.tsx:140]: This is why I need you! 👍🏼 This didn't even come to my mind even though I totally did this in Oxxxx project LOL.

File: app/screens/TalkDetailsScreen/SpeakerPanelScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  paddingHorizontal: spacing.large,
+  paddingBottom: spacing.large,
+}
+
+const $containerSpacing: ViewStyle = {
+  marginBottom: spacing.large,
+}
+
+const $detailsText: TextStyle = {
+  fontSize: 26,
```
Comment (robinheinze) [inline app/screens/TalkDetailsScreen/SpeakerPanelScreen.tsx:109]: Should this be a new Text preset?

Comment (kateinkim) [reply app/screens/TalkDetailsScreen/SpeakerPanelScreen.tsx:109]: I can add that as a preset for sure!

**Review by robinheinze — APPROVED** [review]

**Pull request merged**